src/GafferArnoldUI/DecayVisualiser.cpp

```diff
@@ -233,11 +235,14 @@ Visualisations DecayVisualiser::visualise( const IECore::InternedString &attribu
 		return {};
 	}
 
+	const FloatData *visualiserScaleData = attributes->member<FloatData>( g_glVisualiserScaleString );
+	const float visualiserScale = visualiserScaleData ? visualiserScaleData->readable() : 1.f;
+
 	IECoreGL::GroupPtr result = new IECoreGL::Group();
 
 	for( KnotVector::size_type i = 0; i < knots.size(); ++i )
 	{
-		addKnot( result, knots[i] );
+		addKnot( result, knots[i], visualiserScale );
```

Comment:
Should we update the comment at L249 to say that we've already applied the visualisation scale to the individual indicators? At the moment the comment sort of implies that we don't want to apply it at all.

Reply:
That sounds good, done in https://github.com/GafferHQ/gaffer/pull/5886/commits/8fb1b9e160f88e1372b1e5ef4dfefa228d6740bf